src/Components/test/testassets/TestContentPackage/MyPrompt.cs

```diff
@@ -5,7 +5,7 @@ namespace TestContentPackage
 {
     public static class MyPrompt
     {
-        public static Task<string> Show(IJSRuntime jsRuntime, string message)
+        public static ValueTask<string> Show(IJSRuntime jsRuntime, string message)
```

Comment:
`EventCallback<T>` doesn't have overloads that accept a `ValueTask<T>`. If use you used this function as your callback, you get a method group to delegate conversion error. I think adding overloads to the factory would be enough to make it work. Is there any reason we wouldn't to support ValueTasks?

Reply:
Making any change like that is kinda risky. I'm **very** concerned about doing that without a few previews to see if it causes problems.